test1.js

```diff
@@ -2,4 +2,6 @@
 // two
 // three
 // four
-// five
\ No newline at end of file
+// five
+// six
```

Comment:
can you add 'FIVE' in capitals here`//FIVE`
```suggestion
// FIX
```